nncf/quantization/advanced_parameters.py

```diff
@@ -396,7 +396,7 @@ def apply_advanced_parameters_to_config(
     :param params: Advanced quantization parameters
     :return: advanced quantization parameters as dict in the legacy format
     """
-    config["overflow_fix"] = params.overflow_fix.value
+    config["overflow_fix"] = params.overflow_fix if params.overflow_fix is None else params.overflow_fix.value
```

Comment:
We should have a test for that to prevent further pipeline issues.

Reply:
added